website/docs/compute/get-started.md

```diff
@@ -0,0 +1,212 @@
+# Getting started with Compute
+
+## Installation
+
+To use Compute, you'll need an environment token and the Subo command line 
+tool. The environment token allows Compute to authenticate with 
+our service that manages metadata storage, telemetry, and more.
+
+### Install Subo: the Compute CLI
+
+If you use macOS, you can use [Homebrew](https://brew.sh) to install the 
+`subo` command line tool:
+
+```bash
+brew tap suborbital/subo
+brew install subo
+```
+
+Note: this may take a few minutes! Next, run `subo --version` to ensure the 
+installation was successful. 
+
+To install on Linux (or macOS without Homebrew), you can [download Subo directly](https://github.com/suborbital/subo/releases).
+
+## Generate your token
+You can get a Compute Environment token either with our [environment token generator web app](https://suborbital.network/) or with the `subo` command line tool.
+
+:::caution  
+The domain of your email address should match the `environment` you 
+configure for Compute; i.e., if your email is `laika@example.com`, your 
+Compute environment would be called `com.example`. See [Fully-qualified 
+function names](./customizing-functions/fully-qualified-function-names.md) 
+for more 
+information.
+:::
+
+To create your token, run:
+
+```bash
+subo compute create token <email>
+```
+
+A verification code will be sent to your email address, and the token will be 
+used to authenticate you and link your Compute installation to you.
+
+:::tip
+Subo will print out your token in your terminal, and cache it for use in 
+subsequent steps.
+:::
+
+Keep your environment token safe (store it in your password manager), and 
+only use it when running or installing Compute. If you lose it, just repeat 
+this token generation process.
+
+
+## Run Compute locally
+
+To run Compute locally, you'll use the Subo CLI and [Docker](https://docs.docker.com), both of which must be installed before continuing.
+
+### Prepare your local environment
+
+The Subo CLI will create some files on disk to set up your local deployment,
+so get started by creating a directory somewhere on your filesystem. In the 
+example below, we'll name our directory `suborbital`:
+
+```bash
+mkdir suborbital
+cd suborbital
+```
+
+### Start Compute
+
+Next, use Subo to start your local Compute Core instance. Make sure to do 
+this within the same directory you created above! You can verify that 
+you're in the correct directory by verifying that it contains the 
+`docker-compose.yaml` file.
+
+```bash
+subo compute deploy core --local
+```
+
+You may be asked to enter your Environment Token, and then Subo will use `docker-compose` to launch your Compute Core instance automatically. Compute runs in the background by default. You can use `docker-compose logs -f` to view the logs of the running containers. Run `docker-compose down` to terminate the containers.
```

Comment:
In light of suborbital/subo#344, maybe we should replace references to `docker-compose` with `docker compose`? (Could be handled in a follow-up PR, rather than prolonging this one... but if so, we should make sure it's tracked somewhere.

Reply:
Now tracked here: https://github.com/suborbital/docs/issues/163